### PR TITLE
[css-ui-3] Add missing box-sizing tests

### PR DESCRIPTION
**Pull request by frivoal**
This change is [Reviewable](https://reviewable.io/reviews/w3c/csswg-test/1218)

### CODE REVIEW

**Comment by cbiesinger** [comment]
Why is this in css-ui?

On Feb 27, 2017 3:33 AM, "W3C Sync Bot" <notifications@github.com> wrote:

Automatic validation checks of commit 900b95b
<https://github.com/w3c/csswg-test/commit/900b95b1e44afda3a12efdf2f234b51e9daa1df3>
passed.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/w3c/csswg-test/pull/1218#issuecomment-282657869>, or mute
the thread

**Comment by cbiesinger** [comment]
Nevermind, I just realized box sizing is part of css-ui

File: css-ui-3/box-sizing-027.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿<!DOCTYPE html>
+<title>CSS Basic User Interface Test: box-sizing:border-box used values</title>
```
Comment (fantasai) [inline css-ui-3/box-sizing-027.html]: s/used/resolved/g

**Comment by fantasai** [comment]
r=fantasai with that change :)

**Pull request merged**